### PR TITLE
fix vm verify multi signature bug

### PR DESCRIPTION
**Pull request by tanZiWen**
Fix vm verify multi signature bug

1.When get signature list from vm stack, should use the signature list length for the loop. (there is a mistake in previous code).
2.Verify the signature failure doesn't need to return verify fail, because when verify the multiple signature only need to guarantee that equal the number of the signature and public key is verified successfully.For example, three signatures just need three of the four public key is verified successfully, it is unnecessary verify all successful.

### CODE REVIEW

**Comment by sharpbear** [comment]
Please add useful commit log , for this case , a suggested example like:

Fix vm verify multi signature bug

When get signature list from vm stack, should use the signature list length for the loop. (there is a mistake in previous code)

File: vm/func_crypto.go [hunk 1]
```diff
@@ -70,10 +70,10 @@ func opCheckMultiSig(e *ExecutionEngine) (VMState, error) {
 	fSuccess := true
 
 	for i, j := 0, 0; fSuccess && i < m && j < n; {
-		ver, err := e.crypto.VerifySignature(message, signatures[i], pubkeys[j])
-		if err != nil {
-			return FAULT, err
-		}
+		ver, _ := e.crypto.VerifySignature(message, signatures[i], pubkeys[j])
+		//if err != nil {
```
Comment (dreamfly281) [inline vm/func_crypto.go]: Why the error value be dropped silent,  if it is useless the commending code also not necessary

Comment (tanZiWen) [inline vm/func_crypto.go]: For example: three signatures just need three of the four public key is verified success, it is unnecessary validation of all the success.

Comment (dreamfly281) [inline vm/func_crypto.go]: Jus remove the comments part in that case, and it's better add a explanation above this code to clarify this case

**Pull request closed without merging**